data/secscan_model/secscan_v4_model.py

```diff
@@ -154,11 +154,6 @@ def load_security_information(self, manifest_or_legacy_image, include_vulnerabil
         try:
             report = self._secscan_api.vulnerability_report(manifest_or_legacy_image.digest)
         except APIRequestFailure as arf:
-            try:
-                status.delete_instance()
-            except ReadOnlyModeException:
-                pass
-
```

Comment:
Are there any case where we _should_ be requeuing the manifest for indexing, depending on Clair's response?
e.g a 5XX vs 4XX status code

Reply:
I don't believe so, if the indexing was successful the vulnerability report should be available, if it isn't it's a transient error. @hdonnay can you think of a reason we'd re-queue a manifest when the vulnerability report request is now successful?

Reply:
No, I don't think so. We do need to be able to indicate permanent/transient errors back to quay, but we don't have that yet.